'View More' Button Update

youriwims opened the pull request:
Closes #1037

mmmavis commented on scss/buttons.scss:
```diff
@@ -0,0 +1,126 @@
+.btn {
```
let's move this file to under `/foundation-style` if the code here was copied from foundation site

mmmavis commented:
@youriwims reminder that your PR should be against `master`, not `update-tagline`

<img width="606" alt="image" src="https://user-images.githubusercontent.com/2896608/57278159-9e7efa00-705a-11e9-9dc1-d934c485d7c2.png">

youriwims commented:
@mmmavis yep, I just did that to use the styles from the initial pr so I could keep working while things were getting reviewed. I just rebased and updated it.

youriwims commented:
@mmmavis @natalieworth this is ready for review! 😸

mmmavis commented on scss/foundation-styles/buttons.scss:
```diff
@@ -0,0 +1,126 @@
+.btn {
+  font-family: "Nunito Sans";
+  font-weight: $btn-font-weight;
+  border-radius: 0;
```
If we wanna get rid of `border-radius` for buttons we should do it by removing `$btn-border-radius` in the codebase instead of introducing another override. @natalieworth just to confirm - do we want to reset all buttons's border-radius so they don't have rounded edges?

youriwims replied:
@mmmavis I went ahead and removed this override and instead updated the value of `$btn-border-radius` to `0px`. Removing that variable would involve some variable + mixin clean up which I think are beyond the scope of this small ticket and can be done separately.

mmmavis replied:
Sounds good. We can clean those up as follow up. Can you create a ticket for this as a reminder?

youriwims commented:
@mmmavis I can go ahead and update the other `.btn-outline-info` instance in this pr as well. Thanks for double checking that!
